Update quick-setup.md (#295)
SlimeVR/SlimeVR-Docs-Site: src/quick-setup.md

	![Calibration page](assets/img/quick_calibrate.png)

> 	Please note: Each time you turn your trackers on, they will need to perform this calibration. Make sure to leave them on a flat surface after turning them on each time you want to use the trackers!

Reviewer: ```suggestion
	Please note: Each time you turn your trackers on, they will need to perform this calibration. Make sure to leave them on a flat surface after turning them on each time you want to use your trackers!
```

	![Calibration page](assets/img/quick_calibrate.png)

> 	Please note: Each time you turn your trackers on, they will automatically perform this calibration. make sure to leave them on a flat surface whille turning them on for each use of the trackers!

Reviewer 2: ```suggestion
	Please note: Each time you turn your trackers on, they will need to perform this calibration. Make sure to leave them on a flat surface after turning them on each time you want to use the trackers!
```
Pretty sure it can be any time after boot, but can double check with others

### Spawning trackers

The latest SlimeVR server will automatically assign which SteamVR tracker locations should be used depending on where your SlimeVR trackers are assigned to, which saves the need for deciding which locations needd to be toggled on. However, you can disable this and toggle them manually if you wish to do so

> If you decide to manually decide which steamvr trackers to spawn, you can toggle them in the SteamVR section using the following suggestions:

Reviewer 3: `steamvr` to `SteamVR`

### OSC

If you decide to use OSC Trackers with the Steam version of VRChat, make sure you first disable all of the SteamVR trackers before going to the OSC settings.

![OSC Settings](assets/img/quick_oscSettings.png)

From here you will need to make sure that the network address is set correctly. This will depend on what device you are running VRChat or another program with OSC compatibility on. If you are using this all on the same device, the default `127.0.0.1` should be fine, but if you are using a different device (for example, connecting your trackers to standalone quest), you will need to find the IP address of that device.

Then, you can toggle on which locations that you need using the following suggestions: 

* Lower-Body Set (5 trackers) - Waist and feet.

> * Core Set (5 trackers with one extension) - Chest, waist and feet.

Reviewer: These two options can include knees the same as the other sets.

> * Lower-Body Set (5 trackers) - Waist and feet.
> * Core Set (5 trackers with one extension) - Chest, waist and feet.
> * Enhanced Core Set (5 trackers with three extensions) - Chest, waist, knees and feet.
> * Full-Body Set (7 trackers with three extensions) - Chest, waist, knees, feet and elbows.

Reviewer: Should use digits the same way between counts.
```suggestion
* Lower-Body Set (5 trackers) - Waist and feet.
* Core Set (5 trackers with 1 extension) - Chest, waist and feet.
* Enhanced Core Set (5 trackers with 3 extensions) - Chest, waist, knees and feet.
* Full-Body Set (7 trackers with 3 extensions) - Chest, waist, knees, feet and elbows.
```

### OSC

> If you decide to use OSC use your trackers, make sure you first disable all of the trackers for SteamVR before going to OSC on the settings page.

Reviewer 2: ```suggestion
If you decide to use OSC Trackers with the Steam version of VRChat, make sure you first disable all of the SteamVR trackers before going to the OSC settings.
```

![OSC Settings](assets/img/quick_oscSettings.png)

From here you will need to make sure that the network address is set correctly. This will depend on what device you are running VRChat or another program with OSC compatability on. If you are using this all on the same device, the default 127.0.0.1 should be fine, but if you are using a different device (For example connecting your trackers to standalone quest), you will need to find the ip address of that device.

> Then, you can toggle on which locations that tou neeed using the following suggestions: 

Reviewer 3: `tou neeed` to `you need`
